### ISSUE TITLE
Board/Head Pinout

### ISSUE THREAD

**Issue by mgcrea** [BJORNDEBEN/Fabtotum-Head-cable#2]
By any chance do you have some kind a schematic with the actual pinout of the board, especially the head connector? I'm considering keeping my existing head and using your designed PCB to wire the head to a generic (Duet/SKR) board? But I'm afraid of blowing things up if I wire incorrectly (fan/thermistor/hot-end, etc.).

Thanks!

**Comment by BJORNDEBEN** [comment]
No I dont. I just made an as close as possible copy of the original board. I don't have the pinout. I am also planning on changing the Totumduino board when it breaks down. But as long as it keeps working I will keep using it like this. If you find the correct pinoutput feel free to add it. Or send it to me.

**Comment by mgcrea** [comment]
Looks like we could have guessed them from the published [Totemduino schematics](http://download.fabtotum.com/TOTUMduino%20schematics.PDF) but the link is now dead.

In my case, having played with an Flsun Q5 with TMC2208 silent drivers, the quietness of the printer is really a must have for me now and I want to switch as soon as I can.

I've ordered an [MKS Nano](https://fr.aliexpress.com/item/33013776323.html?spm=a2g0w.12010615.8148356.12.30d5718cufcszX) (same as in the Flsun) to replace the Totumduino.

I've opened an issue there: https://github.com/FABtotum/Head-Development-Kit/issues/2

**Comment by BJORNDEBEN** [comment]
@mgcrea 

I found something on the internet. I will add it to the rest of the files. It are the schematics of the old v1 totumduino.

**Comment by mgcrea** [comment]
Great! Looks like here is the head_connector, will help but I guess we'll have to open the head to check the actual head wiring.

<img width="711" alt="Screenshot 2020-06-20 at 12 09 27" src="https://user-images.githubusercontent.com/108273/85199322-e4def900-b2ee-11ea-8a98-985c6daa1b46.png">

**Comment by BJORNDEBEN** [comment]
Yes indeed. Maybe you can compare and follow the tracs on the original v1 headcable. This way you don't need to open the headboard.

**Comment by mgcrea** [comment]
Found [this repository](https://github.com/imarin2/Protohead_PCB_v4_FAB_24V) with the proper pinout:

![Protohead_v4_24V_rc1_schm](https://user-images.githubusercontent.com/108273/87876758-9484b500-c9da-11ea-8af3-bd1a6a8bed2d.png)

**Comment by BJORNDEBEN** [comment]
Great. This can be very useful

**Comment by mgcrea** [comment]
So I finally made it work but had to cut a trace and rewire your PCB. Not sure how it worked on the original fabtotum but the board I'm now using (Robin Nano) is switching the hotend to ground. And since you've connected all the grounds on the PCB, just having the fan wired would trigger the hotend, that would basically never stop heating. Here are the cuts and tweaks I made. At some point I might redesign a PCB to have JST-style plugs (as I don't need the 20 pins), I also have a small trace alignment issue (but might come from my drilling/tapping of M3 holes).

![P1 P2 P3 P4 P5](https://user-images.githubusercontent.com/108273/88071273-e6ad0e00-cb73-11ea-9287-6f77bbd4a123.JPG)

Had to tweak Marlin to configure the proper thermocouple but everything seems to work OK, last step is to design a BLTouch mount to do automated mesh bed levelling!

**Comment by BJORNDEBEN** [comment]
Great job, happy you got it all working. I think on the original printer they switch the positive side of the hotend. I used sprint layout 6 to design the PCB. I can give you the original file so you don't need to start all over. But I don't think you will be able to open it with other software.

**Comment by nickisghosty** [comment]
are pins 11-12 (blower) the same as pins 47-48 (pressure)?
